.circleci/config.yml

```diff
@@ -145,34 +149,34 @@ jobs:
     steps:
       - checkout
 
-      - android/restore-gradle-cache:
-          cache-prefix: << parameters.gradle-cache-prefix >>
       - android/restore-build-cache:
           cache-prefix: << parameters.build-cache-prefix >>
 
+      - run:
+          name: List availabe emulator images
+          command: sdkmanager --list
+
       - android/start-emulator-and-run-tests:
           #   It should match the name seen in the "sdkmanager --list" output
-          system-image: system-images;android-33;google_apis;x86_64
+          system-image: system-images;android-29;default;x86
```

Comment:
I'm keeping the same system-image we are using for the other Android Edge extensions. This [CircleCI blog post on the Android Emulator](https://support.circleci.com/hc/en-us/articles/360000028928-Testing-with-the-Android-emulator-on-CircleCI) states that test issues have been observed with the higher API level images which use Google APIs and it is suggested to use lower API images.